public/resources/remix/index.js

```diff
@@ -6,20 +6,32 @@
 
   var gaTrackingId = 'UA-68630113-1';
 
-  function customizeScroll($) {
-    var bodyEl = $("body");
-    var detailsBar = $(".remix-details-bar");
-    var detailsBarHeight = 64;
-    var currentPadding = parseInt(bodyEl.css("padding-top"));
-    bodyEl.css("padding-top", currentPadding + detailsBarHeight);
-
-    $(window).on("scroll", function() {
-      var scrollDistance = $(this).scrollTop();
-      if(scrollDistance >= detailsBarHeight) {
-        detailsBar.addClass("scrolled");
-      } else {
-        detailsBar.removeClass("scrolled");
-      }
+  function setupBar(){
+    var isTouchDevice = 'ontouchstart' in document.documentElement;
+    var detailsBar = document.querySelector(".details-bar");
```

Comment:
The rest of this file uses jquery, can you update this function to do the same, so we don't mix styles?